dm/templates/cloudbuild/cloudbuild.py.schema

```diff
@@ -412,12 +412,12 @@ definitions:
           The directory to run the build in (relative to the source root).
           If a step's dir is specified as an absolute path, the value is
           ignored for that step's execution.
-      revision:
```

Comment:
This needs to stay 'revision'. If you call the property same as the replacements inside the oneOfs it fails

Reply:
I can not call it, as this "revision" field is not supported in gcp type.